src/usage_metrics/models.py

```diff
@@ -507,3 +508,196 @@
     Column("remote_ip_full_location", String),
     Column("partition_key", String),
 )
+
+
+core_eel_hole_logs = Table(
+    "core_eel_hole_logs",
+    usage_metrics_metadata,
+    Column(
+        "insert_id",
+        String,
+        primary_key=True,
+        comment="A unique identifier for the log entry.",
+    ),
+    Column(
+        "timestamp",
+        DateTime,
+        comment="The time the event described by the log entry occurred.",
+    ),
+    Column(
+        "text_payload",
+        String,
+        comment="Data provided to the logger in a text format. For the viewer, this includes the redirects generated by a user when logging in.",
+    ),
+    Column(
+        "event",
+        Enum("search", "hit", "duckdb_preview", "duckdb_csv", "log_in"),
+        comment="What type of event the log is recording (e.g., exporting to CSV, searching for a record).",
+    ),
+    Column(
+        "query",
+        String,
+        comment="What did a user type into the search box? This logs periodically, so multiple partial search queries may be logged as someone is typing.",
+    ),
```

Comment:
non-blocking but are these queries duplicative for the various param filters? could these actually be two or three separate tables? a query table and a filter table?